source/partials/application/_hosting.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if application.production_hosted_on == "aws" %>
+<strong><%= application.hosting_name %></strong>, on the <strong><%= application.aws_puppet_class %></strong> machine class
```

Comment:
A very minor thing, but the `if`, `elsif` and `else` code should be indented once

Reply:
Ah yes - originally this file was parsed as Markdown, so the indentation caused it to be treated as a code block. I think I can safely fix up the indentation now.